app/graphql/resolvers/submissions_resolver.rb

```diff
@@ -21,14 +22,22 @@ def valid?
   end
 
   def run
-    base_submissions.where(conditions).order(sort_order)
+    submissions = base_submissions.where(conditions).order(sort_order)
+
+    submissions.map do |submission|
+      if admin? || partner? || submission.draft?
+        submission
+      else
+        submission.as_json(properties: :short)
```

Comment:
We don't need the `.order(sort_order)` anymore?

Reply:
Do you mean that the order of the submissions will be broken?

Reply:
I said that based on the diff.
`base_submissions.where(conditions).order(sort_order)`